CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 2.1.0 (2023-02-17)
```

Comment:
This definitely should be a major revision!

Reply:
Is this already published as 2.1?

Reply:
Nope, just getting it ready. But yeah it could be a 3.0.